src/layouts/InterfaceLayout/containers/NFTManagerContainer/containers/NftDetails/NftDetails.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="crypto-kitties-details">
+    <div class="crypto-kitties-details-content-container">
+      <back-button @click.native="goBack" />
+      <div class="grid-container">
+        <div class="kitty-image">
+          <img :src="nft.image" />
+        </div>
+        <div class="kitty-text">
+          <h3>{{ $t('dapps.sendMy', { value: selectedTitle }) }}</h3>
+          <p>#{{ nft.token }}</p>
+          <div class="address-input-container">
+            <address-selector title="To Address" @toAddress="prepareTransfer" />
+            <div class="send-button-container">
+              <standard-button
+                :button-disabled="!isValidAddress"
+                :options="sendButton"
+                @click.native="transfer"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import { ERC721, KittyCore } from '../../abis/';
+
+import { Misc, Toast } from '@/helpers';
+import InterfaceContainerTitle from '@/layouts/InterfaceLayout/components/InterfaceContainerTitle';
+import SmallBackButton from '@/layouts/InterfaceLayout/components/SmallBackButton';
+import DropDownAddressSelector from '@/components/DropDownAddressSelector';
+import StandardButton from '@/components/Buttons/StandardButton';
+
+export default {
+  components: {
+    'interface-container-title': InterfaceContainerTitle,
+    'back-button': SmallBackButton,
+    'address-selector': DropDownAddressSelector,
+    'standard-button': StandardButton
+  },
+  props: {
+    selectedTitle: {
+      type: String,
+      default: 'Error'
+    },
+    nft: {
+      type: Object,
+      default: function() {
+        return {};
+      }
+    },
+    selectedContract: {
+      type: String,
+      default: 'Error'
+    },
+    nftConfig: {
+      type: Object,
+      default: function() {
+        return {};
+      }
+    }
+  },
+  data() {
+    return {
+      toAddress: '',
+      tokenContract: {},
+      ERC721tokenContract: {},
+      cryptoKittiesContract: {},
+      cryptoKittiesConfig: '',
+      sendButton: {
+        title: 'Send',
+        buttonStyle: 'green',
+        helpCenter: true,
+        noMinWidth: true,
+        fullWidth: true
+      }
+    };
+  },
+
+  computed: {
+    ...mapState([
+      'account',
+      'gasPrice',
+      'web3',
+      'network',
+      'linkQuery',
+      'online'
+    ]),
+    isValidAddress() {
+      if (this.toAddress !== '') {
+        return Misc.isValidENSorEtherAddress(this.toAddress);
+      }
+      return false;
+    }
+  },
+  watch: {},
+  mounted() {
+    this.ERC721tokenContract = new this.web3.eth.Contract(ERC721);
+  },
+  methods: {
+    prepareTransfer(toAddress) {
+      this.toAddress = toAddress;
+      this.ERC721tokenContract.options.address = this.nft.contract;
+    },
+    buildData() {
+      if (
```

Comment:
cant we just use `transferFrom` which is common in all 721 including crypto kitties